### ISSUE TITLE
CSV Reading: Can only parse 'delimiter' argument on file_name, not file_stream etc

### ISSUE THREAD

**Issue by garfunkel** [pyexcel/pyexcel#76]
Reading a CSV file object/stream with a custom delimiter currently appears to be impossible. It seems this can only be done when file_name is provided, not file_stream.

Exception thrown:
UnknownParameters: Please check if there were typos in function parameters: {'file_stream': , 'delimiter': '\t'}. Otherwise unrecognized parameters were given.

**Comment by chfw** [comment]
Could you please share more information? 

I cannot reproduce the issue. Please take a look at the test case and suggest what to change in order to reproduce it here.

**Comment by garfunkel** [comment]
:man_facepalming: 

My bad. After looking over the pyexcel/core.py source code, I realised that my call to get_sheet() was incorrect. When loading in from a file stream, one must provide file_type arg as well as the delimiter. When I did this, it works as expected. Sorry.

Was doing:

get_sheet(file_stream = f, delimiter = delim)

Should be doing:

get_sheet(file_stream = f, file_type = "csv", delimiter = delim)

I think source documentation can be clarified to avoid this in future. I'll make a pull request.
